src/core/bittorrent/torrenthandle.h

```diff
@@ -178,9 +178,46 @@ namespace BitTorrent
         qlonglong pieceLength() const;
         qlonglong wastedSize() const;
         QString currentTracker() const;
-        QString actualSavePath() const;
-        QString savePath() const;
-        QString rootPath() const;
+
+        // 1. savePath() - the path where all the files and subfolders of torrent are stored (as always).
+        // 2. rootPath() - absolute path of torrent file tree (save path + first item from 1st torrent file path).
+        // 3. contentPath() - absolute path of torrent content (root path for multifile torrents, absolute file path for singlefile torrents).
+        //
+        // These methods have 'actual' parameter (default to false) which allow to get current or target path variant.
+        //
+        // Examples.
+        // Suppose we have three torrent with following structures and save path `/home/user/torrents`:
+        //
+        // Torrent A (multifile)
+        //
+        // subdirs/
+        //   file1
+        //   file2
+        //   file3
+        //
+        //
+        // Torrent B (singlefile)
+        //
+        // subdirs/
+        //   file1
+        //
+        //
+        // Torrent C (singlefile)
+        //
+        // file1
+        //
+        //
+        // Results:
+        // |   |          rootPath          |           contentPath            |
+        // |---|----------------------------|----------------------------------|
+        // | 1 | /home/user/torrents/subdir | /home/user/torrents/subdir       |
```

Comment:
Nitpick here:
| 1 | /home/user/torrents/subdir | /home/user/torrents/subdir       |
to 
| **A** | /home/user/torrents/subdir | /home/user/torrents/subdir       |